EIPS/eip-2780.md

```diff
@@ -10,54 +10,107 @@ created: 2020-07-11
 ---
 
 ## Abstract
-Reduce the intrinsic cost of a transaction to `16,360` gas.
+Reduce the intrinsic cost of a transaction from `21,000` to `7,000` gas.
```

Comment:
How was this value 7000 selected? There does not seem to be an explanation how it was derived, while the previous version was very clear about that.

Reply:
My understanding is that @uri-bloXroute plans to do some tests to determine if lower intrinsic gas costs will increase the uncle rate, and `7,000` is only a starting point. In that case, he is taking an empirical approach -- where as the original draft was an attempt to "deduce" the fair cost a tx based on known resource costs. 

While I don't prefer the strictly empirical approach, I also don't have the time to champion this and want Uri to feel empowered to experiment and gather data. Hopefully the result of that will be beneficial for everyone!

Reply:
@axic - @lightclient is correct.
I don't believe the approach of calculating the cost of running a Tx on the EVM (under the worst-case assumption that all transactions are sent to a new account) is the right one. Why does it matter how much it would have cost?

My argument is:
- Sending Tx on chain became very expensive due to DeFi
- It would be valuable to allow for as many Tx as possible, but we must consider the negative externalities
- We should enable as low gas cost as possible (I believe we can do under 7,000) as long as we do not  create such externalities (or they are negligible)
- I consider the implications on uncle-rate (to be verified on testnet), state-size, and gas-token. I should probably add measurements of block validation on commodity PCs, to show they are not in danger of being pushed off the network.

Does that rationale make sense to you? I wanted to use this EIP as a starting point for a discussion with the different devs.